File: Problems/Solved/Easy/2798.js
/**
 * @param {number[]} hours
 * @param {number} target
 * @return {number}
 */
var numberOfEmployeesWhoMetTarget = function (hours, target) {
    let result = 0;
    let l = 0;
    let r = hours.length - 1;
    while (l < r) {
        if (hours[l] >= target) result++;
        if (hours[r] >= target) result++;
        l++;
        r--;
    }
    if (l === r && hours[l] >= target) result++;
    return result;
};

File: Problems/Solved/2798.rs
pub fn number_of_employees_who_met_target(hours: Vec<i32>, target: i32) -> i32 {
    let mut l = 0;
    let mut r = hours.len() - 1;
    let mut result = 0;
    while l < r {
        if hours[l] >= target {
            result += 1;
        }
        if hours[r] >= target {
            result += 1;
        }
        l += 1;
        r -= 1;
    }
    if r == l && hours[l] >= target {
        result += 1
    };
    result
}
